pkg/openshift/rosa/accountroles.go

```diff
@@ -136,6 +142,10 @@ func (r *Provider) getAccountRoles(ctx context.Context, prefix, version string)
 			continue
 		}
 
+		if strings.HasPrefix(prefix, "fr") {
```

Comment:
Is this standard in some way? Like does rosa automatically create with this prefix or is this an arbitrary prefix we are defining? I think it would be cleaner to determine `isFedRamp` based on a value set on the `Provider` field that is set through the initialization of the provider.